pixi.js.d.ts

```diff
@@ -659,32 +665,32 @@ declare namespace PIXI {
         static TEMP_MATRIX: Matrix;
 
     }
-    export class ObservablePoint {
 
-        constructor(cb: () => any, scope?: any, x?: number, y?: number);
+    class PointLike {
```

Comment:
That's the goal of the Typescript interface keyword though - not an actual object, but a group of expected properties and methods.